src/documentation/docs/components/NewsDetail.mdx

```diff
@@ -13,47 +15,92 @@ import PropsTableRow from "@/components/internal/Documentation/PropsTable/compon
-  <PropsTableRow value="tags" type="Array">
-      Specify which tags should be displayed. Required attributes are:
-      <ul>
-        <li><span class="code">label</span> Label of the tag</li>
-      </ul>
-
-  </PropsTableRow>
   <PropsTableRow value="teaser" type="String">
-    Specify which teaser should be displayed above the picture.
+    Text between headline and picture.
   </PropsTableRow>
   <PropsTableRow value="date" type="String">
-      Specify which date should be displayed.
+      Date when the article was written
+  </PropsTableRow>
+  <PropsTableRow value="tags" type="Array">
+      Tags under the article. Possible attributes are:
+      <ul>
+        <li><span class="code">label</span>string - Required - Label of the tag</li>
+        <li><span class="code">id</span>number | string -  Id of the tag</li>
+      </ul>
   </PropsTableRow>
   <PropsTableRow value="author" type="String">
-      Specify which author should be displayed.
+    Author of the article
   </PropsTableRow>
   <PropsTableRow value="handleTagClick" type="Function">
       You can pass in an optional callback that will be triggered, when a tag
       is clicked.
     </PropsTableRow>
+  <PropsTableRow value="returnText" type="String">
+    The text that will be displayed on the return button under the article.
+  </PropsTableRow>
   <PropsTableRow value="handleReturnClick" type="Function">
         You can pass in an optional callback that will be triggered, when the return button
             is clicked.
   </PropsTableRow>
-  <PropsTableRow value="socialText" type="String">
-    The text that will be displayed above the social media icons in the footer.
-  </PropsTableRow>
-  <PropsTableRow value="returnText" type="String">
-    The text that will be displayed on the return button in the footer.
+  <PropsTableRow value="social" type="Object">
+    Information about sociallinks. Possible attributes are:
+    <ul>
+          <li><span class="code">title</span>string - Title of the social area</li>
+          <li><span class="code">items</span>Object - social media items. Possible attributes are
+            <ul>
+              <li><span class="code">title</span>string - Title of the social element</li>
+              <li><span class="code">id</span>number | string - id of the social element</li>
+              <li><span class="code">options</span>Object - additional information of the social element</li>
+            </ul>            
+          </li>
+          <li><span class="code">handleItemClick</span>Function - Title of the social area</li>
+    </ul>
   </PropsTableRow>
 </PropsTable>
 
+## Slots
+
+<SlotsTable>
+  <SlotsTableRow value="social" prop="SocialArea">
+    Specify your custom HTML for the social area. These attributes you get as value:
+    <ul>
+          <li><span class="code">title</span>string - Title of the social area</li>
+          <li><span class="code">items</span>Object - social media items. Possible attributes are
```

Comment:
There's something wrong with the nested lists on chrome. They do not render correctly. Please redesign the whole list.